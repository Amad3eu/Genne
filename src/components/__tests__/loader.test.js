import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Loader from '../loader';

describe('Loader', () => {
  const component = shallow(<Loader />);

  it('renders correctly', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
