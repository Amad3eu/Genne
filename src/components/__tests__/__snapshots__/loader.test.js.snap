// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  className="loader"
>
  <span>
    ↓
  </span>
  <span>
    ↓
  </span>
  <span>
    ↓
  </span>
  <span>
    ↓
  </span>
  <span>
    ↓
  </span>
</div>
`;
